Makefile

```diff
@@ -0,0 +1,2 @@
+docs/commcare-cloud/commands/index.md : src/commcare_cloud/* src/commcare_cloud/*/* src/commcare_cloud/*/*/*
+	manage-commcare-cloud make-docs > docs/commcare-cloud/commands/index.md
```

Comment:
Are there any features missing from something like sphinx that we gain by creating our own auto doc infrastructure?